util/misc.go

```diff
@@ -430,7 +434,26 @@ type SequenceTable interface {
 // LoadTLSCertificates loads CA/KEY/CERT for special paths.
 func LoadTLSCertificates(ca, key, cert string) (tlsConfig *tls.Config, err error) {
 	if len(cert) == 0 || len(key) == 0 {
-		return
+		tempStoragePath := config.GetGlobalConfig().TempStoragePath
+		cert = tempStoragePath + "/cert.pem"
+		key = tempStoragePath + "/key.pem"
+		err = createTLSCertificates(cert, key)
+		if err != nil {
+			logutil.BgLogger().Warn("TLS Certificate creation failed", zap.Error(err))
+			return
+		}
+		go func() {
+			for {
+				select {
+				case <-time.After(time.Hour * 24 * 30):
```

Comment:
Nice solution!

Should we also randomise the reloading over a day, so not all the TiDB nodes reloads at the same time, but spreads it out a bit? (Only a suggestion, patch approved)